packages/main/src/Input.ts

```diff
@@ -131,6 +134,8 @@ type SuggestionScrollEventDetail = {
 	scrollContainer: HTMLElement;
 }
 
+const Suggestions = getFeature<typeof InputSuggestions>("InputSuggestions");
```

Comment:
This could be a problem, because the feature might not be imported yet. In the old code, this call was made later in the lifecycle of the component (when `dependencies` were accessed for the first time). We should investigate it

Reply:
We found solution for this